app/src/main/java/com/x8bit/bitwarden/data/auth/datasource/disk/model/PendingAuthRequestJson.kt

```diff
@@ -16,4 +18,10 @@ data class PendingAuthRequestJson(
 
     @SerialName("PrivateKey")
     val requestPrivateKey: String,
+
+    @SerialName("AccessCode")
+    val requestAccessCode: String,
+
+    @SerialName("Fingerprint")
```

Comment:
Since we are making a breaking change here anyways, can we make these all camel case